tests/beaker_tests/cisco_acl/test_ace.rb

```diff
@@ -13,307 +13,136 @@
-    'established'   => 'true',
-    'packet_length' => 'range 80 1000',
-    'time_range'    => 'my_range',
-    'redirect'      => 'Ethernet1/1,Ethernet1/2,port-channel1',
+tests[:seq_10_v4] = {
+  title_pattern:  'ipv4 beaker 10',
+  manifest_props: {
+    action:   'permit',
+    proto:    'tcp',
+    src_addr: '1.2.3.4 2.3.4.5',
+    dst_addr: '9.9.0.4/32',
   },
 }
 
-tests['ipv6_seq_10'] = {
-  desc:           '2.1 ipv6 beaker_6 seq 10',
-  title_pattern:  'ipv6 beaker_6 10',
-  manifest_props: "
-    action          => 'deny',
-    proto           => 'tcp',
-    src_addr        => 'any',
-    dst_addr        => 'any',
-  ",
-  resource:       {
-    'action'   => 'deny',
-    'proto'    => 'tcp',
-    'src_addr' => 'any',
-    'dst_addr' => 'any',
+tests[:seq_10_v6] = {
+  desc:           'IPv6 Seq 10',
+  title_pattern:  'ipv6 beaker6 10',
+  platform:       'n(3|7|9)k', # TBD: N5k addrs display as 0001:0001::0001/128
+  manifest_props: {
+    action:   'permit',
+    proto:    'tcp',
+    src_addr: '1:1::1/128',
+    dst_addr: '1:1::2/128',
   },
 }
 
-tests['ipv6_seq_20'] = {
-  desc:           '2.2 ipv6 beaker_6 seq 20',
-  title_pattern:  'ipv6 beaker_6 20',
-  manifest_props: "
-    remark          => 'test remark'
-  ",
-  resource:       {
-    'remark' => 'test remark'
+tests[:seq_20_v4] = {
+  title_pattern:  'ipv4 beaker 20',
+  manifest_props: {
+    action:        'deny',
+    proto:         'tcp',
+    src_addr:      'any',
+    src_port:      'eq 40',
+    dst_addr:      'any',
+    dst_port:      'range 32 56',
+
+    established:   'true',
+    packet_length: 'range 80 1000',
+    precedence:    'flash',
+    redirect:      'Ethernet1/1,Ethernet1/2,port-channel1',
+    tcp_flags:     'ack syn fin',
+    time_range:    'my_range',
+
+    # TBD: ttl is currently broken on NX platforms
+    # ttl:           '127',
   },
 }
-
-tests['ipv6_seq_30'] = {
-  desc:           '2.3 ipv4 beaker_6 seq 30',
-  title_pattern:  'ipv6 beaker_6 30',
-  manifest_props: "
-    action        => 'permit',
-    proto         => 'tcp',
-    src_addr      => 'any',
-    src_port      => 'eq 40',
-    dst_addr      => 'any',
-    dst_port      => 'range 32 56',
-    tcp_flags     => 'ack syn fin',
-    dscp          => 'af12',
-    packet_length => 'range 80 1000',
-    time_range    => 'my_range',
-    log           => true,
-  ",
-  resource:       {
-    'action'        => 'permit',
-    'proto'         => 'tcp',
-    'src_addr'      => 'any',
-    'src_port'      => 'eq 40',
-    'dst_addr'      => 'any',
-    'dst_port'      => 'range 32 56',
-    'tcp_flags'     => 'ack syn fin',
-    'dscp'          => 'af12',
-    'packet_length' => 'range 80 1000',
-    'time_range'    => 'my_range',
-    'log'           => 'true',
+tests[:seq_20_v6] = tests[:seq_20_v4].clone
+tests[:seq_20_v6][:title_pattern] = 'ipv6 beaker6 20'
+
+tests[:seq_30_v4] = {
+  desc:           'IPv4 Seq 30',
+  title_pattern:  'ipv4 beaker 30',
+  platform:       'n(3|9)k',
```

Comment:
Do you still need the platform key here, if you are using the unsupported_properties method to also filter the properties?